### PR TITLE
Updated UI of task-detail section #7954

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

File: ng-app/app/views/task.html [hunk 1]
```diff
@@ -5,87 +5,106 @@
...
-            ng-show="comment.objectType == 'project.work.history'"/>
-          <div class="md-list-item-text" ng-if="comment.objectType == 'project.work.history'">
-            <span class="progress-change" ng-if="comment.new_progress_state">
-              <span>{{ comment.previous_progress_state }}</span>
-              <i class="mdi mdi-arrow-right-bold"></i>
-              <span>{{ comment.new_progress_state }}</span>
-            </span>
-            <span ng-if="comment.new_assignee">Assigned to {{ comment.new_assignee.displayName }}</span>
-            <p marked="comment.comment"><p>
+  <div layout="row" class="task-comment-desc">
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: `comment-container`

File: ng-app/app/styles/main.css [hunk 2]
```diff
@@ -250,3 +245,21 @@ md-list-item a {
...
+  min-height: 44px;
+  background: #B8B6B6;
+}
+.text-black{
+  color: #000;
+}
+.task-comment-desc .md-avatar{
+  vertical-align: middle;
+}
+.task-comment-desc .toolbar-font{
```
Comment (GauravButola) [inline ng-app/app/styles/main.css]: `.comment-container md-toolbar`

File: ng-app/app/styles/main.css [hunk 3]
```diff
@@ -250,3 +245,21 @@ md-list-item a {
...
+.custom-toolbar {
+  height: 44px;
+  max-height: 44px;
+  min-height: 44px;
+  background: #B8B6B6;
+}
+.text-black{
+  color: #000;
+}
+.task-comment-desc .md-avatar{
```
Comment (GauravButola) [inline ng-app/app/styles/main.css]: `vertical-align-middle`

File: ng-app/app/styles/main.css [hunk 4]
```diff
@@ -250,3 +245,21 @@ md-list-item a {
...
+.img-circle {
+  border-radius: 50%;
+}
+.custom-toolbar {
+  height: 44px;
+  max-height: 44px;
+  min-height: 44px;
+  background: #B8B6B6;
+}
+.text-black{
```
Comment (GauravButola) [inline ng-app/app/styles/main.css]: `.text-black {`

File: ng-app/app/styles/main.css [hunk 5]
```diff
@@ -250,3 +245,21 @@ md-list-item a {
 .cursor-pointer {
   cursor: pointer;
 }
+.img-circle {
+  border-radius: 50%;
+}
+.custom-toolbar {
```
Comment (GauravButola) [inline ng-app/app/styles/main.css]: `comment-container`

File: ng-app/app/views/task.html [hunk 6]
```diff
@@ -5,87 +5,106 @@
...
+    <md-whiteframe class="md-whiteframe-z1" layout-margin flex>
+      <md-toolbar class="custom-toolbar">
+        <div class="md-toolbar-tools md-subhead custom-toolbar">
+          <div layout-margin>
+            <img gravatar-src="task.assigned_to.email" gravatar-size="38" class="md-avatar img-circle" alt="{{task.assigned_to.displayName}}" />
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: `task.created_by`

File: ng-app/app/views/task.html [hunk 7]
```diff
@@ -5,87 +5,106 @@
...
           </div>
-
-          <div ng-if="comment.objectType == 'timesheet.line'">
-            Worked for {{ comment.displayName }}
+          <div class="text-black toolbar-font">
+            {{ task.created_by.displayName }} Opened on {{ task.create_date | date:'medium' }}
           </div>
-          <md-divider ng-if="comment.objectType == 'project.work.history'"></md-divider>
-        </md-list-item> <!-- /Task com ment -->
+          <div flex></div>
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: `span`

File: ng-app/app/views/task.html [hunk 8]
```diff
@@ -5,87 +5,106 @@
...
-          <div class="md-list-item-text" ng-if="comment.objectType == 'project.work.history'">
-            <span class="progress-change" ng-if="comment.new_progress_state">
-              <span>{{ comment.previous_progress_state }}</span>
-              <i class="mdi mdi-arrow-right-bold"></i>
-              <span>{{ comment.new_progress_state }}</span>
-            </span>
-            <span ng-if="comment.new_assignee">Assigned to {{ comment.new_assignee.displayName }}</span>
-            <p marked="comment.comment"><p>
+  <div layout="row" class="task-comment-desc">
+    <md-whiteframe class="md-whiteframe-z1" layout-margin flex>
```
Comment (GauravButola) [inline ng-app/app/views/task.html:15]: Add comment `<!-- Task description -->`

File: ng-app/app/views/task.html [hunk 9]
```diff
@@ -5,87 +5,106 @@
...
-        <md-list-item ng-if="commentObj.comment" class="unsaved-comment cursor-pointer">
-          <img gravatar-src="user.email" class="md-avatar" alt="{{ user.displayName }}" />
-          <div class="md-list-item-text">
-            <span class="progress-change" ng-if="task.progress_state != commentObj.progress_state">
-              <span>{{ task.progress_state }}</span>
-              <i class="mdi mdi-arrow-right-bold"></i>
-              <span>{{ commentObj.progress_state }}</span>
-            </span>
-            <p marked="commentObj.comment"><p>
+  <div layout="column" class="task-comment-desc">
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: Add comment `<!-- Comment description -->`

File: ng-app/app/views/task.html [hunk 10]
```diff
@@ -5,87 +5,106 @@
...
-          Worked for {{ commentObj.hours }} hours
-        </div> <!-- /Draft comment -->
-        <md-divider></md-divider>
-        <md-progress-linear ng-if="savingComment" md-mode="indeterminate"></md-progress-linear>
-
-      </md-list>
+          <div class="text-black toolbar-font">
+            {{ comment.updatedBy.displayName }} commented on {{ comment.create_date | date:'medium'}}
+          </div>
+           <div flex></div>
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: `span`

File: ng-app/app/views/task.html [hunk 11]
```diff
@@ -5,87 +5,106 @@
...
+          <span class="progress-change" ng-if="comment.new_progress_state">
```
Comment (GauravButola) [inline ng-app/app/views/task.html:48]: Remove this and css as well

File: ng-app/app/views/task.html [hunk 12]
```diff
@@ -5,87 +5,106 @@
...
+            <span>{{ comment.previous_progress_state }}</span>
+            <i class="mdi mdi-arrow-right-bold"></i>
+            <span>{{ comment.new_progress_state }}</span>
+          </span>
+          <span ng-if="comment.new_assignee">Assigned to {{ comment.new_assignee.displayName }}</span>
+        </div>
+      </md-toolbar>
+      <div layout-padding layout-margin>
+        <div class="text-black" ng-if="comment.objectType == 'project.work.history'">
+
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: remove blank line

File: ng-app/app/views/task.html [hunk 13]
```diff
@@ -5,87 +5,106 @@
...
+      </div>
     </md-whiteframe>
+  </div>
+  <div ng-if="comment.objectType == 'timesheet.line'">
+    Worked for {{ comment.displayName }}
+  </div>
+  <md-divider ng-if="comment.objectType == 'project.work.history'"></md-divider>
+     <!-- /Task com ment -->
 
+      <!-- Draft comment -->
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: deindent

File: ng-app/app/views/task.html [hunk 14]
```diff
@@ -5,87 +5,106 @@
...
+          
+          <p marked="comment.comment"><p>
+        </div>
+      </div>
     </md-whiteframe>
+  </div>
+  <div ng-if="comment.objectType == 'timesheet.line'">
+    Worked for {{ comment.displayName }}
+  </div>
+  <md-divider ng-if="comment.objectType == 'project.work.history'"></md-divider>
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: Remove

File: ng-app/app/views/task.html [hunk 15]
```diff
@@ -5,87 +5,106 @@
...
+      <md-tooltip>Comment not saved yet. Press 'Comment' To update.</md-tooltip>
+    </md-list-item>
+    <div ng-if="commentObj.hours">
+      Worked for {{ commentObj.hours }} hours
+    </div> <!-- /Draft comment -->
+    <md-divider></md-divider>
+    <md-progress-linear ng-if="savingComment" md-mode="indeterminate"></md-progress-linear>
     <md-whiteframe class="md-whiteframe-z1" layout layout-padding layout-margin>
-      <form name="commentForm" ng-submit="submitComment()" layout="column" flex>
+        <form name="commentForm" ng-submit="submitComment()" layout="column" flex>
```
Comment (GauravButola) [inline ng-app/app/views/task.html]: deindent

**Comment by GauravButola** [comment]
LGTM

**Comment by gautampanday** [comment]
nereid project-material

File: ng-app/app/styles/main.css [hunk 16]
```diff
@@ -250,3 +245,52 @@ md-list-item a {
...
+  z-index: 0;
+}
+.line-vertical .md-whiteframe-z1 {
+  background: #fff;
+  position: relative;
+  z-index: 1;
+  margin-bottom: 30px;
+}
+
+.line-vertical .mdi-checkbox-multiple-blank-outline {
```
Comment (GauravButola) [inline ng-app/app/styles/main.css]: Add a class instead of adding it to specific icon

**Comment by GauravButola** [comment]
LGTM

**Pull request merged**